samples/msgext-ai-sentiment-analysis/nodejs/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+	"recommendations": [
```

Comment:
Please check if we need to push this file?

Reply:
fixed